src/agent/include/register.hpp

```diff
@@ -1,8 +1,16 @@
 #pragma once
 
+#include <http_client.hpp>
```

Comment:
Is this include necessary?

Reply:
After a few changes, now it's needed to use `http_client::AuthenticateFunctionType`